src/components/TheTransactionList.vue

```diff
@@ -9,7 +9,13 @@
     >
       <TransactionListRecent />
     </div>
-    <div class="modal is-active" v-if="showModal">
+    <div
+      class="modal"
+      :class="[
+        getCurrentModal.component == this.$options.name ? 'is-active' : '',
+      ]"
+      @click="datePickerShow = false"
```

Comment:
there is a nice way to write this: 
```javascript
{
 'is-active': getCurrentModal.component == this.$options.name
}
```

Reply:
hmm, what is this `is-active` class ? You have it in other modals, and do not change it. Is that normal ?

Reply:
done